### DISCUSSION TITLE
How to display billions in google sheets

### DISCUSSION THREAD

**Discussion by Philhazell89** [FlowmapBlue/FlowmapBlue#66] [Q&A]
I'm relatively new to google sheets and I'm trying to write billions into the google template sheet for FlowMapBlue but the number is invalid? How do I resolve this issue/work around it?
![Screenshot (342)](https://user-images.githubusercontent.com/107126661/172635085-77a5459c-52f8-4e80-abca-94d1aeeb2da0.png)

**Comment by ilyabo** [comment]
Hi Phil,

Yes, it should be just numbers without letters and separators. Google Sheets will automatically show separators unless you change the cell format to "Plain text".

**Reply by Philhazell89** [reply]
Thank you, that resolved the issue. I have another question. How do I change/delete "incoming trips, outcoming trips, internal & round trips" on each location? I have noticed that if I delete/alter msg.total cells in properties, it changes messaging on each arm but not the messaging on the origin/destination
![Screenshot (345)](https://user-images.githubusercontent.com/107126661/172854282-380fa2d1-2bd2-4dc5-983d-547195db230d.png)

**Reply by ilyabo** [reply]
Not sure if I understood you correctly, but you probably need to change `msg.locationTooltip.*`:

<img width=300 src=https://user-images.githubusercontent.com/351828/172855249-fb2a329f-5f56-4afa-8ff8-7903094414bb.png>

The `msg.totalCount.*` props are for what's shown at the bottom of the info-box:

<img width=300 src=https://user-images.githubusercontent.com/351828/172855563-b59122f2-58a0-4bfc-9b7e-433a2c487bac.png>

**Reply by Philhazell89** [reply]
I've deleted those cells, but they're still appearing on the map.

**Reply by ilyabo** [reply]
Ah, I see. No you cannot hide them completely.
